tests/test_ingest.py

```diff
@@ -318,6 +318,48 @@ def test_mtx_bundle_argument_validation(self):
 
         self.assertRaises(ValueError, self.setup_ingest, args)
 
+    def test_ingest_loom(self):
+        """Ingest Pipeline should extract and transform loom files
+        """
+
+        args = [
+            "--study-accession",
+            "SCP1",
+            "--file-id",
+            "1234abc",
+            "ingest_expression",
+            "--taxon-name",
+            "Homo Sapiens",
+            "--taxon-common-name",
+            "human",
+            "--ncbi-taxid",
+            "9606",
+            "--genome-assembly-accession",
+            "GCA_000001405.15",
+            "--genome-annotation",
+            "Ensemble 94",
```

Comment:
Also updated in mock data.

```suggestion
            "Ensembl 94",
```

Reply:
Addressed in f708656